op-service/txmgr/txmgr.go

```diff
@@ -1036,14 +1043,24 @@ func (m *SimpleTxManager) SuggestGasPriceCaps(ctx context.Context) (*big.Int, *b
 		return nil, nil, nil, nil, fmt.Errorf("tip is too high: %v, cap:%v", tip, maxTipCap)
 	}
 
+	// Check minimum tip cap (for blob txs). Note that we check the same limits for blob txs as for non-blob txs.
+	if minTipCap != nil && blobTipCap.Cmp(minTipCap) == -1 {
+		m.l.Warn("Suggested blobTipCap is lower than the configured min tip cap for blob txs", "minTipCap", minTipCap, "blobTipCap", blobTipCap)
+		// We do not enforce the min tip cap for the suggested blob tip cap on purpose, as we are generally overpaying with the statically set minTipCap.
+		// blobTipCap = new(big.Int).Set(minTipCap)
```

Comment:
Could we just say "the min tip cap does not apply to the blob tip cap"? Rather node leave commented-out code in place. O since this whole block does nothing, we could just remove L1046-L1051?